plugin/resurrect/pane_tree.lua

```diff
@@ -94,16 +94,41 @@ local function insert_panes(root, panes)
 		return nil
 	end
 
-	root.cwd = root.pane:get_current_working_dir().file_path
-	if is_windows then
-		root.cwd = root.cwd:gsub("^/([a-zA-Z]):", "%1:")
-	end
-	root.process = root.pane:get_foreground_process_name()
-	if pub.get_shell_process(root.process) then
-		root.text = root.pane:get_lines_as_escapes(root.pane:get_dimensions().scrollback_rows)
+	local domain = root.pane:get_domain_name()
+	if not wezterm.mux.get_domain(domain):is_spawnable() then
+		wezterm.log_warn("Domain " .. domain .. " is not spawnable")
+		wezterm.emit("resurrect.error", "Domain " .. domain .. " is not spawnable")
 	else
-		root.text = {}
+		if not root.pane:get_current_working_dir() then
+			root.cwd = ""
+		else
+			root.cwd = root.pane:get_current_working_dir().file_path
+			if is_windows then
+				root.cwd = root.cwd:gsub("^/([a-zA-Z]):", "%1:")
+			end
+		end
+
+		if string.sub(domain, 1, 3) == "SSH" then
+			root.domain = domain
+			-- we cannot get process information or lines from SSH domains
+			-- see https://wezfurlong.org/wezterm/config/lua/pane/get_foreground_process_name.html
+			root.process = ""
+			root.text = {}
+		else
+			if domain ~= "local" then
+				-- TODO: handle WSL, UNIX and TLS domains
+				wezterm.log_warn("Domain " .. domain .. " is not currently supported by resurrect.wezterm")
```

Comment:
Added an issue which we can refer to, as it can be misinterpreted as this is not something which will ever be supported
```suggestion
				wezterm.log_warn("Domain " .. domain .. " is not currently supported by resurrect.wezterm, please see: https://github.com/MLFlexer/resurrect.wezterm/issues/40")
```